cinema/views.py

```diff
@@ -76,7 +103,14 @@
         return MovieSerializer
 
 
-class MovieSessionViewSet(viewsets.ModelViewSet):
+class MovieSessionViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.DestroyModelMixin,
+    viewsets.GenericViewSet,
+):
     queryset = (
         MovieSession.objects.all()
```

Comment:
The annotation for `tickets_available` in `MovieSessionViewSet` is correct, but ensure that the logic aligns with the business requirements for ticket availability.